conda.recipe/meta.yaml

```diff
@@ -31,6 +31,7 @@ requirements:
 
 test:
   requires:
+    - avro # [py27]
```

Comment:
This recipe affects building conda packages, which isn't actually used in testing.  We would run into this issue upon releasing.